dist/tools/pyterm/pyterm

```diff
@@ -126,9 +128,10 @@ class SerCmd(cmd.Cmd):
         self.json_regs = dict()
         self.init_cmd = []
         self.load_config()
-        self.fmt_str = None
-        if not self.fmt_str:
+        if self.fmt_str == None:
```

Comment:
I'm not sure I understand.
If formatter is not set (i.e. no formatter is specified via command line), these lines should use either `default_fmt_str` or the one found in the config file.